run-server.sh

```diff
@@ -195,15 +234,21 @@ pipenv run ./manage.py runapscheduler | tee -a /var/www/logs/scheduler.log &
 sleep 5
 
 echo -e "\n[""$BLUE JOB""$NC""] Starting Apache"
-pipenv run ./manage.py runmodwsgi --user www-embark --group sudo \
---host "$BIND_IP" --port="$HTTP_PORT" --limit-request-body "$FILE_SIZE" \
+WSGI_CMD="pipenv run ./manage.py runmodwsgi --user www-embark --group sudo \
```

Comment:
should we move this to an array? See https://google.github.io/styleguide/shellguide.html#arrays